types/office-js-preview/index.d.ts

```diff
@@ -34519,28 +34528,28 @@ declare namespace Excel {
         readonly custom: Excel.CustomPropertyCollection;
         /**
          *
-         * Gets or sets the author of the workbook.
+         * The author of the workbook.
```

Comment:
`Gets or sets` or `Represents`
```suggestion
         * Represents the author of the workbook.
```